tests/hooks/test_postgres_hook.py

```diff
@@ -24,6 +24,54 @@
 from tempfile import NamedTemporaryFile
 
 from airflow.hooks.postgres_hook import PostgresHook
+from airflow.models import Connection
+
+
+class TestPostgresHookConn(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        self.connection = Connection(
+            login='login',
+            password='password',
+            host='host',
+            schema='schema'
+        )
+
+        self.db_hook = PostgresHook()
+        self.db_hook.get_connection = mock.Mock()
+        self.db_hook.get_connection.return_value = self.connection
+
+    @mock.patch('airflow.hooks.postgres_hook.psycopg2.connect')
+    def test_get_conn(self, mock_connect):
+        self.db_hook.get_conn()
+        assert mock_connect.call_count == 1
```

Comment:
```suggestion
        assert mock_connect.call_count == 1
```

Reply:
ah, forgot to pull that out. good catch.